### PR TITLE
Define User Data for Mounting EFS/NFS Volume

### PR DESCRIPTION
**Pull request by goruha**
## What
* Define `user_data` script for mounting NFS filesystem

## Why
* Easily mount EFS filesystems

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,18 @@
+resource "null_resource" "default" {
```
Comment (osterman) [inline main.tf]: Use `tf_label`

File: user_data.sh [hunk 2]
```diff
@@ -0,0 +1,72 @@
+
```
Comment (osterman) [inline user_data.sh:1]: This should include the eb env file (and not assume it's already been imported)

**Review by osterman — CHANGES_REQUESTED** [review]
See inline comments.

File: user_data.sh (hunk 2 above)
Comment (goruha) [reply user_data.sh:1]: It imported for user data by default

**Review by osterman — APPROVED** [review]

**Pull request merged**